src/identifier/mod.rs

```diff
@@ -266,30 +242,51 @@ fn is_file(name: &str) -> bool {
     }
 }
 
-fn is_valid_filter(configs: &Identifier, regex_data: &Data) -> bool {
+/// Validation filter
+#[derive(Debug)]
+pub enum ValidationFilter {
+    DataLowerRarity,
+    DataHigherRarity,
+    DataInTags,
+    DataInExclude,
+    Good,
+}
```

Comment:
the name of the enum and the variants can be improved to better match the purpose they are serving.